PWG/EMCAL/macros/AddTaskRejectMCBackground.C

```diff
@@ -0,0 +1,55 @@
+AliEmcalRejectMCBackground* AddTaskRejectMCBackground(
```

Comment:
This macro should in principle just call the static add function you implemented in the analysis task. Here you basically duplicate the code.